src/modules/mount/main.py

```diff
@@ -299,6 +300,10 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
             else:
                 mount_option_no_subvol = get_mount_options(fstype, mount_options, partition)
             mount_option = f"subvol={s['subvolume']},{mount_option_no_subvol}"
+
+            # If the subvolume is "" don't add any subvol= arguments
+            if not s['subvolume']:
+                mount_option = mount_option_no_subvol
```

Comment:
```suggestion
            # Only add subvol= argument if we are not mounting the entire filesystem
            if s['subvolume']:
                 mount_option = f"subvol={s['subvolume']},{mount_option_no_subvol}"
            else:
                mount_option = mount_option_no_subvol
```

Reply:
Yep, makes sense, made that change trying it on my usb stick now.

Reply:
Yes this seems to have worked here. Marking resolved.